cmd/common/utils/flags.go

```diff
@@ -131,3 +133,22 @@ func AddRuntimesSocketPathFlags(command *cobra.Command, config *RuntimesSocketPa
 		"Podman Unix socket path",
 	)
 }
+
+func AddRegistryAuthVariablesAndFlags(cmd *cobra.Command, authOptions *oci.AuthOptions) {
+	// Flag inspired by https://github.com/containers/common/blob/cac40138f7e3c2b29ca32e64348535516bf6aa51/pkg/auth/cli.go#L48
+	cmd.Flags().StringVar(
+		&authOptions.AuthFile,
+		"authfile",
+		oci.DefaultAuthFile,
+		"path of the authentication file. This overrides the REGISTRY_AUTH_FILE environment variable",
+	)
+	viper.BindPFlag("registry.auth_file", cmd.Flags().Lookup("authfile"))
+	viper.BindEnv("registry.auth_file", "REGISTRY_AUTH_FILE")
```

Comment:
Our environment variables should be documented and prefixed with `IG_`. The prefix could be automatically configured with:
```
viper.SetEnvPrefix("ig")
```

Reply:
In general I agree.

For this flag and environment variable i explicitly used the same as other programs so it can be easily interoperable ([podman-login](https://docs.podman.io/en/stable/markdown/podman-login.1.html), [skopeo-login](https://github.com/containers/skopeo/blob/main/docs/skopeo-login.1.md), [buildah-login](https://www.mankier.com/1/buildah-login))

WDYT about keeping it that way?

Reply:
Ok as long as it's documented

Reply:
I documented it.